lib/local_authorities_url_importer.rb

```diff
@@ -0,0 +1,93 @@
+require 'csv'
+
+class LocalAuthoritiesUrlImporter
+  CONTACTS_LIST_URL = "http://local.direct.gov.uk/Data/local_authority_contact_details.csv"
+
+
+  def self.import_urls
+    fh = fetch_data
+    begin
+      new(fh).import
+    ensure
+      fh.close
+    end
+  end
```

Comment:
I changed this importer to work less like the csv importers in Publisher (where it came from) and more like the other ones in this app that work on json.  We're no longer creating temp files which tidies this up.